.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: lint
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why only on master?

Reply:
I did not think about it. I'll update the PR for all branches